ThemeHelper.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * ThemeHelper Class Doc Comment.
+ *
+ * Include this in the theme file to access the helper methods.
+ *
+ * @category Class
+ * @package gent_base
+ * @author Bart Delrue
+ */
+class ThemeHelper {
+
+  /**
+   * @param string $key
+   *   The view mode key.
+   *
+   * @return mixed
+   *   Returns the view mode if found.
+   */
+  public static function isViewMode($key) {
+    return \Drupal::service('entity_type.manager')
```

Comment:
I'm not entirely sure, but I think you should use dependency injection here instead of calling this service with \Drupal::